extract_unique_sensors_01.py

```diff
@@ -0,0 +1,33 @@
+from urllib.request import urlopen
+
+url = ('http://monctl.devm.camlab.kat.ac.za/kat/doc/manuals/sensors/SensorList.txt')
+try:
+    sensors_data = urlopen(url)
+except Exception:
+    filename = ('/home/lebukani/Downloads/SensorList.txt')
+    with open(filename) as lines:
+        sensors_data = lines.readlines()
+    
+def extract_all_sensors(lines): 
+    """Filters the sensors_data and returns only a list of sensors.The function accepts lines as a string and nothing else."""
+    candidate_lines = []
```

Comment:
Did you open and read the content of this link?
https://docs.google.com/document/d/1aZoIyR9tz5rCWr2qJKuMTmKp2IzHlFjrCFrpDDHFypM/edit#heading=h.t1m6y2gpknsq

Reply:
No, I just read it now.

Reply:
Apply :laughing: